LocalPayment/src/test/java/com/braintreepayments/api/LocalPaymentLauncherUnitTest.kt

```diff
@@ -67,20 +65,23 @@ class LocalPaymentLauncherUnitTest {
 
     @Test
     fun `handleReturnToAppFromBrowser on BrowserSwitchResult returns result`() {
-        val browserSwitchResult = Mockito.mock(
-            BrowserSwitchResult::class.java
+        val browserSwitchResultInfo = Mockito.mock(
```

Comment:
Is it possible to use Mockk here instead of Mockito?

Reply:
Yes fixed 👍